sci-libs/dlib/dlib-19.15.recipe

```diff
@@ -0,0 +1,99 @@
+SUMMARY="Numerical and networking C++ library"
+DESCRIPTION="Dlib is a modern C++ toolkit containing machine learning \
+algorithms and tools for creating complex software in C++ to solve real \
+world problems."
+HOMEPAGE="http://dlib.net/"
+COPYRIGHT="2003-2018 Davis E. King"
+LICENSE="Boost v1.0"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/dclib/dlib-$portVersion.tar.bz2"
```

Comment:
Here's a mirror: http://dlib.net/files/dlib-19.15.tar.bz2

Come straight from the homepage :)